vendor/bat-native-ads/include/bat/ads/ads_history.h

```diff
@@ -23,7 +23,8 @@ struct ADS_EXPORT AdsHistory {
 
   enum class FilterType {
     kNone = 0,
-    kConfirmationType
+    kConfirmationType,
```

Comment:
Question only: can we/should we change `kConfirmationType` to be more specific to it's use, given the new `kAdConversionConfirmationType` below?

Reply:
Agreed so will raise another ticket as want to keep this PR bare minimum as may be uplifted